apis/accessing-endpoints.md

```diff
@@ -0,0 +1,42 @@
+# Accessing API Endpoints
+
+Kubecost Cloud has select endpoints available for API access. The default API access endpoint will be `api.app.kubecost.com`. Obtaining access involves generating an API key value, which is then used to generate a personal access token, which is used to retrieve your team ID value. The token and team ID value will then allow you to make queries.

```

Comment:
The team ID is already available as a request header. Getting access to that is not related to the personal access token. 

```suggestion
Kubecost Cloud has select endpoints available for API access. The default API access endpoint will be `api.app.kubecost.com`. Obtaining access involves generating an API key value, which is then used to generate a personal access token. The token, together with your team ID, will then allow you to make queries.

```

Reply:
Bit confused, the doc I referenced shows the command used to retrieve the team ID requires you to input the personal access token. Assuming this interpretation is incorrect?

Reply:
That's correct, you need the personal access token to access eg. https://auth.app.kubecost.com/api/user, which returns team information. However, you can get the team ID through other means as well, without needing the personal access token -- for example, if you inspect any page in KC Cloud, you can find the team ID in any of the request headers.

I've got a frontend [PR in review](https://github.com/kubecost/kubecost-frontend/pull/1181) to add a new "Team Info" section at the bottom of the Settings page, where we would display the current team name and ID. This would be by far the easiest way to obtain it. Once that's merged I'll let you know!

Reply:
Ok, this has been merged. See the screenshot in the PR description

Reply:
Awesome addition, thanks for doing that. I'll replace this section with a simpler means of finding the Team ID